pkg/metadata/sandbox.go

```diff
@@ -115,11 +115,6 @@ func (s *sandboxStore) Get(sandboxID string) (*SandboxMetadata, error) {
 	if err != nil {
 		return nil, err
 	}
-	// Return nil without error if the corresponding metadata
-	// does not exist.
-	if data == nil {
-		return nil, nil
```

Comment:
just making sure there's no chance for data to be a nil here? otherwise you'll get the json unmarshal error